ssl_inspector.py

```diff
@@ -1,32 +1,57 @@
 #! /usr/bin/python3
```

Comment:
- Powtarzającym się problemem w całym kodzie jest przemieszanie kodu odpowedzialnego za wyświetlanie na ekranie lub input z kodem wykonującym różne operacje jak odczyt z bazy danych czy plików. O ile wykorzystanie wzorca Model View Controller jest przesadą w tak małym programie to rozdzieliłbym te operacje. Pozwoliłoby to np łatwo zmienić sposób wyświetlania na np HTML, czy aplikację okienkową PySide, lub narzędzie CLI.

Np:

```
def main_menu_fun(...):
...
            elif current_row == 1:  #   Load database
                db_path = load_from_database_input(...)
                conn = open_database(db_path)
                if conn is None:
                    show_error(f"Cannot open database: {db_path}")
...
```

Gdzie `open_database` zastępuje `load_from_database`, `show_error` jest nową funkcją do wypisywania błędów na ekran.

- Nie pisałbym własnej implementacji menu tylko skorzystał z biblioteki menu dostępnej w ncurses: https://pypi.org/project/curses-menu/
- Pozwolił na podanie ścieżki do katalogu z certyfikatami albo do bazy danych jako parametr linii komend
- Wykorzystałbym klasy. Pierwsza to mogłaby być UI - odpowiedzialna za wyśiwtlanie danych, druga była by strukturą danych z metodami pozwalającymi na odczyt danych z bazy, plików oraz może zapis.
- Baza jest bardzo prosta ale jako bonus można by się nauczyć SQLAlchemy